testsuite/src/test/java/org/jboss/galleon/userchanges/test/UserChangesAfterInstallTestCase.java

```diff
@@ -124,7 +124,6 @@ protected DirState provisionedHomeDir() {
         return newDirBuilder()
                 .addFile("prod1/p1.txt", "prod1 p1")
                 .addFile("prod1/p2.txt", "user")
-                .addFile("prod1/p2.txt.glnew", "prod1 p2")
```

Comment:
Could these files be modified in a way that create a conflict just to preserve these tests? They seem to be valuable as they are. It'd be better to add new tests for the feature you are introducing.

Reply:
I think this would require creating an updated `prod1` FP and installing the updated package on top of `prod2` - similarly to `UserChangesAfterUpdatePlanTestCase` 
Alternatively `prod2` could override the `prod1/p2.txt` file, but that use case is already captured in `common.txt`.

Reply:
Ah, if it's already covered then we are good. Thanks.

Reply:
@spyrkob could you please confirm the rest of the existing tests are also not undermined by these changes?